docs-documenter/src/buffering.md

```diff
@@ -0,0 +1,23 @@
+# Buffering
+
+As mentioned in the section on [transformations](./tfminterface.md), you can implement [`apply!`](#) as an inplace version of [`apply`](#) to support buffered transformations. Usually, the result of a regular `apply` can be used as a buffer. You may write
```

Comment:
References like this need to be mapped from Pollen to Documenter syntax. In general, you need to replace ``[`func`](#)`` with ``[`func`](@ref)``.

Reply:
yeah you are right, I just missed a few here and there. I'll proceed with fixing links and replacing the directory